keybase/config.go

```diff
@@ -139,6 +139,13 @@ func (c *config) SetUpdateAuto(auto bool) error {
 	return c.save()
 }
 
+// For overriding the current Auto setting
+func (c *config) OverrideAuto() bool {
```

Comment:
I would make a new boolean called `AutoOverride` in config and then check that in the updater (e.g. if Auto || AutoOverride). So that we don't accidentally change/save this.